commands.js

```diff
@@ -2007,6 +2007,7 @@ exports.commands = {
 	blacklist: function (target, room, user) {
 		if (!target) return this.parse('/help blacklist');
 		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (target === 'show') return this.parse('/showblacklist');
```

Comment:
We can disable the user to prevent it from causing an issue in practice, but it's inconsistent to do something like that.